vhdl_lang/src/config.rs

```diff
@@ -102,6 +103,8 @@ impl LibraryConfig {
 
 impl Config {
     pub fn from_str(string: &str, parent: &Path) -> Result<Config, String> {
+        let env_vars: HashMap<String, String> = std::env::vars().collect();
```

Comment:
This creates a copy of all environment variables. I assume that this is in order to test the code? I would prefer it if this wasn't the case, but if it's too much of a hassle you can leave it as-is

Reply:
That's true, but that's just how `std::env::vars()` works: it returns an iterator over `String` pairs. There's no way to avoid the copy that implies.

The alternatives are:
* Do this only once at plugin startup and store it in a `lazy_static` or similar, but that's annoying.
* Call `std::env::vars()` for each substitution separately, but that makes even more copies.

Reply:
Did you consider [std::env::var](https://doc.rust-lang.org/std/env/fn.var.html) for this use-case?

Reply:
I forgot about that one! To keep testing working maybe we can take an `Fn(K) -> Result<String, VarError>` as a parameter and then pass `std::env::var` for the real use case?

Reply:
That sounds like a nice solution!